packages/lib/src/components/Login/components/LoginForm/LoginForm.tsx

```diff
@@ -90,6 +90,7 @@ export function LoginForm({
                 render={({ field }) => (
                   <TextInput
                     {...field}
+                    aria-label={t('loginView.form.email') as string}
```

Comment:
I think it's not necessary to set an extra aria-label if we already have set a label and the name is already available. Same for the other inputs.

Reply:
The thought behind is, to make the app more accessible via ARIA properties. Since `findByRole` is the prio A query for finding elements, it requires an `aria-label` property which I found a good side-effect. According to the [react testing library docs on priority](https://testing-library.com/docs/queries/about/#priority):

>This should be your top preference for just about everything. There's not much you can't get with this (if you can't, it's possible your UI is inaccessible).

Reply:
@PhilKsr I pushed a new commit. The cannot do it by role, it looked like it worked, but it doesn't, it just skipped the implementation in line 50 or so and jumped directly to the error in line 155. I got rid of the aria label since it's set automatically via Mantine and queried the password input via its placeholder value. Please give it another review 🚀